src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/OmniSharpTests.cs

```diff
@@ -21,7 +21,8 @@ public class OmniSharpTests : SmokeTests
 
     public OmniSharpTests(ITestOutputHelper outputHelper) : base(outputHelper) { }
 
-    [SkippableTheory(Config.ExcludeOmniSharpEnv, skipOnTrue: true)]
+    // Disable the test when run offline because it requires downloading the OmniSharp tarball.
+    [SkippableTheory(new string[] { Config.ExcludeOmniSharpEnv, Config.OfflineEnv }, skipOnTrue: true)]
```

Comment:
I was under the impression this was going to be an ephemeral change.  By this I mean exclude the omnisharp tests from the offline runs.  Because of that, I wasn't expecting the need to add offline infra rather CI would just set ExcludeOmniSharpEnv.  When the OmniSharp issue is fixed, I don't see offline being checked anywhere else so it would be dead code to cleanup.